### ISSUE TITLE
PHP 7 non development release

### ISSUE THREAD

**Issue by luislobo** [bitnami/bitnami-docker-php-fpm#38]
Hi,

I've checked the PHP version, and it says
```
root@5f989418b7c5:/app# php -v
PHP 7.0.6 (cli) (built: May  3 2016 18:07:54) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies
    with Zend OPcache v7.0.6-dev, Copyright (c) 1999-2016, by Zend Technologies
root@5f989418b7c5:/app# 
```
I'm concerned about Zend OPcache *v7.0.6-dev*

Are there any release version, instead of a dev one?

Thanks

**Comment by prydonius** [comment]
Hey @luislobo,

That's currently the version of OPcache included in the PHP 7.0.6 stable release (https://github.com/php/php-src/blob/php-7.0.6/ext/opcache/ZendAccelerator.h#L30). You could check some of the [other PHP versions](https://hub.docker.com/r/bitnami/php-fpm/tags/) to see if they include a stable version of OPcache, but unfortunately this is the only release we have for the 7.x series.

Hope this helps!

**Comment by luislobo** [comment]
OK! great, thank you for the reference and information!
